### PR TITLE
[14.0][MIG] migration of module auditlog

### PR DESCRIPTION
**Pull request by alan196**
### Proposed changes

I've migrated the module auditlog and its functionalities from Odoo 13.0 to 14.0. I've checked if all unit tests are still running.

### Types of changes

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Migration Update

### Checklist

- [x] I have read the CONTRIBUTING doc
- [x] I have signed the CLA
- [x] Lint and unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have updated the necessary documentation

### CODE REVIEW

**Review by tonygalmiche — APPROVED** [review]
LGTM : Code review and functional review with runbot

image

**Comment by PieterPaulussen** [comment]
@alan196 Thanks for the migration.
I was wondering if you would mind taking a look at a bug report I made on this module a while ago since you've been working on this module. 

https://github.com/OCA/server-tools/pull/1766

**Review by PieterPaulussen — APPROVED** [review]

File: auditlog/views/auditlog_view.xml [hunk 1]
```diff
@@ -0,0 +1,241 @@
...
+            </form>
+        </field>
+    </record>
+    <record model="ir.ui.view" id="view_auditlog_rule_tree">
+        <field name="name">auditlog.rule.tree</field>
+        <field name="model">auditlog.rule</field>
+        <field name="arch" type="xml">
+            <tree
+                decoration-info="state == 'draft'"
+                decoration-bf="state == 'subscribed'"
```
Comment (ivantodorovich) [inline auditlog/views/auditlog_view.xml:61]:
```suggestion
```

I would remove this decoration to align to other decorations across odoo.
Normally, "validated/posted" records have no decoration.

**Review by ivantodorovich — APPROVED** [review]
Small comment,
otherwise LGTM :+1: 

code review & functional test
Thanks @alan196 !

**Review by nbessi — APPROVED** [review]

**Comment by CasVissers-360ERP** [comment]
@dreispt 
Merge should be restarted?

**Pull request merged**